### PR TITLE
[swss]: Update swss.sh script to clean up specific db when start

### PR DESCRIPTION
**Pull request by stcheng**
This script shall not flush all the entries in the state database
when it starts up, since there are entries maintained and written
by other processes outside this docker.

The issue we noticed was that the portchannel states are cleaned
up after teamsyncd writes the entries into the database, which
causes the IPs failed to be configured because intfmgrd considers
the portchannels are not ready yet.

Signed-off-by: Shu0T1an ChenG <shuche@microsoft.com>

### CODE REVIEW

File: files/scripts/swss.sh [hunk 1]
```diff
@@ -77,8 +89,8 @@ start() {
     if [[ x"$WARM_BOOT" != x"true" ]]; then
         /usr/bin/docker exec database redis-cli -n 0 FLUSHDB
         /usr/bin/docker exec database redis-cli -n 2 FLUSHDB
-        /usr/bin/docker exec database redis-cli -n 5 FLUSHDB
```
Comment (jipanyang) [inline files/scripts/swss.sh]: #define APPL_DB         0
#define ASIC_DB         1
#define COUNTERS_DB     2
#define LOGLEVEL_DB     3
#define CONFIG_DB       4
#define PFC_WD_DB       5
#define FLEX_COUNTER_DB 5
`#define STATE_DB        6`

**Comment by stcheng** [comment]
@jipanyang yes teamd docker shall be responsible for cleaning up the state of LAG
I'll generate separate pull request against that which requires some more changes and testings

**Comment by lguohan** [comment]
how fast is the function?

**Comment by qiluo-msft** [comment]
Suggest use LUA script to optimize, ideally one transaction for one key pattern.

**Comment by stcheng** [comment]
I tested this and surprisingly this is very fast.

**Comment by lguohan** [comment]
can you time it? do you have a measurement? this is critical to fast reboot timing.

**Comment by stcheng** [comment]
LUA script is much faster. The redis command solution will cost roughly 2-3 seconds, while LUA script will consume less than 1 second.

File: files/scripts/swss.sh [hunk 2]
```diff
@@ -62,6 +62,20 @@ function wait_for_database_service()
     done
 }
 
+# This function cleans up the state database entries that are dirty across
+# two restarts
+function clean_up_state_database()
```
Comment (qiluo-msft) [inline files/scripts/swss.sh]: Maybe useful to provide some paramters.
```
function clean_up_database()
{
    DB=$1
    PATTERNS=$2
```

**Pull request merged**